admin/controllers/redirects/controller.php

```diff
@@ -0,0 +1,19 @@
+<?php
+defined('CMSPATH') or die; // prevent unauthorized access
+
+// router
+
+$segments = CMS::Instance()->uri_segments;
+$content_type_filter=null;
+
+if (sizeof($segments)==1) {
+	$view = 'all';
+}
+else {
+	$view = $segments[1];
+}
```

Comment:
allows `/admin/redirects/yolo` to work, which then 500's since the folder doesnt exist